string device_name
string device_state
string description
